spec/support/factory_girl.rb

```diff
@@ -27,4 +27,11 @@
     outdoor_seating false
     pet_friendly true
   end
+
+  factory :review do
+    sequence(:title) { |n| "This bar is awesome #{n}" }
+    rating 5
+    body "The food, drinks and service are amazing"
+    bar_id 3
```

Comment:
you need a user association here too:

```ruby
factory :review do
  seq...
  rating 5
  body "blah"
  
  bar
  user
end
```

That way when you generate a review factory, it generates a bar and user factory for THAT review.